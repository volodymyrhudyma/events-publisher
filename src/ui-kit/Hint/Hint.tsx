import React, { ReactNode } from 'react';
import { Wrapper } from './styled/Hint.styled';

interface IHintProps {
	children: ReactNode;
}

const Hint = ({ children }: IHintProps) => <Wrapper>{children}</Wrapper>;

export default Hint;
