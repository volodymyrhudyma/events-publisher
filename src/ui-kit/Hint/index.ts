import Component from './Hint';

export default Component;
