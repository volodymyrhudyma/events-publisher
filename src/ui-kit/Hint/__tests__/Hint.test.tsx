import React from 'react';
import { shallow } from 'enzyme';
import Hint from '../Hint';

describe('Hint component', () => {
  it('should render correctly', () => {
    const component = shallow(<Hint>Hint</Hint>);
    expect(component).toMatchSnapshot();
  });
});
