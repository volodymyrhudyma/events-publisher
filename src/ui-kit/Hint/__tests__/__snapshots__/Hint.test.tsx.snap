// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hint component should render correctly 1`] = `
<styled.div>
  Hint
</styled.div>
`;
